salesagility/SuiteCRM
# Closed #1764 by adding translations to SuiteSpots

## Description
Closed #1764 
This fix adds the localisation of labels for Suite Spots to allow for all of the labels and messages to be translated appropriately.
This has been tested with the French language pack.

As per https://github.com/c3js/c3/issues/1566, there is also a fix for the mouse over in Chrome (to stop throwing errors).

## Motivation and Context
This allows for the language labels and data to be translated appropriately for Suite Spots.

## How To Test This
Install a language pack and change language.  The spots module should reflect this on all of the labels and options.

## Types of changes

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)

### Final checklist


- [ X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [X ] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.
